lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -87,7 +88,7 @@ def main():
         output_file = f"{output_dir}/calibration.Run{run}.0000.hdf5"
         log_file = f"{output_dir}/log/calibration.Run{run}.0000.log"
         print(f"\n--> Output file {output_file}")
-        if os.path.exists(output_file) and ff_calibration is 'yes':
+        if os.path.exists(output_file) and ff_calibration == 'yes':
```

Comment:
why 'yes' instead of a `Bool`?

Reply:
I have no idea. I just fixed the comparison operator.